code/datums/trading/trade.dm

```diff
@@ -145,8 +145,13 @@
 		var/is_wanted = 0
 		if(is_type_in_list(offer,wanted_items))
 			is_wanted = 1
-		if(blacklisted_trade_items && blacklisted_trade_items.len && is_type_in_list(offer,blacklisted_trade_items))
-			return 0
+		if(blacklisted_trade_items && blacklisted_trade_items.len)
+			if(ishuman(offer))
+				var/mob/living/carbon/human/A = offer
+				if(A.species in blacklisted_trade_items)
```

Comment:
`blacklisted_trade_items` is a list of datums, not a list of instances. You prob. want `is_type_in_list()` here.